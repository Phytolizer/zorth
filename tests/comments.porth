include "std.porth"

//comment1
// comment 2
//////// lots of slashes

// string literal with `//` is fine
"hello, // world\n" stdout write // allowed here as well
