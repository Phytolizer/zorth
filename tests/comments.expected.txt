: returncode 0
: stdout 16
Hello, // World

: stderr 0

